content/blog/2019-12-03-posits/index.md

```diff
@@ -0,0 +1,262 @@
++++
+title = "The Cult of Posits"
+extra.author = "Dietrich Geisler & Edwin Peguero"
++++
+
+Computers are incapable of representing arbitrary real numbers exactly.
+This is due to two intractable facts of real numbers:
+- Uncountably Infinite Domain: There are an infinite number of real numbers.
+- Uncountably Infinite Precision: Some real numbers require infinite precision.
+ 
+Since computers use a finite number of bits, computer architects must settle on capturing a finite number of real numbers at a finite level of precision.
+This is done by fixing a mapping between bit patterns and real numbers called a *representation*.
+A representation makes a tradeoff between the quantity of representable numbers and the level of precision.
+
+## The Floating Point Representation
+
+The *floating point* representation is the most widely used.
+Numbers are written in the form `(-1^s) * 1.m * 2^e`, where `1.m`, the *mantissa*, and `e`, the *exponent*, are fractional and integer binary values, respectively, and `s` is a single bit denoting the sign of the represented number.
```

Comment:
FWIW it's possible to use proper LaTeX math in these posts if you want (see the README).

Reply:
installed zola / katex on windows to implement / test out LaTeX math